modules/infra/eks/users.tf

```diff
@@ -14,3 +16,23 @@ resource "aws_eks_access_policy_association" "admin" {
     type = "cluster"
   }
 }
+
+#########################################
+# non admin users
+
+resource "aws_eks_access_entry" "user" {
+  for_each      = toset(var.eks_user_arns)
+  cluster_name  = aws_eks_cluster.main.name
+  principal_arn = each.value
+}
+
+resource "aws_eks_access_policy_association" "user" {
+  for_each      = toset(var.eks_user_arns)
+  cluster_name  = aws_eks_cluster.main.name
+  policy_arn    = "arn:aws:eks::aws:cluster-access-policy/AmazonEKSAdminPolicy"
```

Comment:
It seems like there are two places we're defining policy, directly attached to the user here and attached to a group below. Am I reading this PR right?

Having two places to modify something can cause issues when new people come into a repo, find the one place and assume it's the only place. Is there any way to join these two policy pieces so their either in the same file or part of the same group? Also disregard if I'm just not understanding :D

Reply:
Yeah you're right, the policies get applied in different places but it's necessary because they associate a policy to a user _and_ to the cluster. We can't apply this policy to the user in the `bootstrap` terraform because the cluster "doesn't exist yet" in there. Here we consume the users as output of the bootstrap terraform and then layer the cluster specific policies on top. It's a good callout though, I'll add some comments to explain.

(In general I think I need to be doing more explanatory commenting, please keep asking pointed questions to help with that!)